ENYO-2754: Remove pressed class for preventing pressed state.

kbs12e opened the pull request:
In sunstone's GridListSample, remove pressed class in setupItem() for preventing
pressed state when page is re-generated because of reordering.

https://jira2.lgsvl.com/browse/ENYO-2754
Enyo-DCO-1.1-Signed-off-by: Bongsub Kim bongsub.kim@lgepartner.com

kbs12e committed:
ENYO-2754: Remove pressed class for preventing pressed state.

In sunstone's GridListSample, remove pressed class in setupItem() for preventing
pressed state when page is re-generated because of reordering.

https://jira2.lgsvl.com/browse/ENYO-2754
Enyo-DCO-1.1-Signed-off-by: Bongsub Kim <bongsub.kim@lgepartner.com>

kbs12e committed:
ENYO-2754: Remove pressed class when setupItem is called.

Remove pressed class when setupItem is called in sunstone's ListSamples

https://jira2.lgsvl.com/browse/ENYO-2754
Enyo-DCO-1.1-Signed-off-by: Bongsub Kim <bongsub.kim@lgepartner.com>

seunghoh commented on sunstone/src/GridListSample.js:
```diff
@@ -93,14 +93,15 @@ module.exports = kind({
 		this.$.reorderTile.setSelected(this.$.list.isSelected(i));
 	},
-	setupItem: function(inSender, inEvent) {
+	setupItem: function (inSender, inEvent) {
```
We usually don't add space after 'function'

kbs12e committed:
ENYO-2754: Remove white space to follow coding conventions

https://jira2.lgsvl.com/browse/ENYO-2754
Enyo-DCO-1.1-Signed-off-by: Bongsub Kim <bongsub.kim@lgepartner.com>
